cmd/kubelet/app/options/options.go

```diff
@@ -140,6 +165,15 @@ func (f *KubeletFlags) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&f.NodeIP, "node-ip", f.NodeIP, "IP address of the node. If set, kubelet will use this IP address for the node")
 
 	fs.StringVar(&f.ProviderID, "provider-id", f.ProviderID, "Unique identifier for identifying the node in a machine database, i.e cloudprovider")
+
+	fs.StringVar(&f.CertDirectory, "cert-dir", "/var/run/kubernetes", "The directory where the TLS certs are located. "+
```

Comment:
always set up defaults in NewKubeletServer(), and echo current value here as the default (see existing flags for an example)